source/iface/platform.rst

```diff
@@ -36,7 +38,8 @@ A platform contains 1 or more SYCL devices, or a host device.
 
 Construct a SYCL platform instance.
 
-The default constructor creates a host platform. When passed a
+The default constructor create a SYCL platform instance that is a copy of the platform which
+contains the device returned by default_selector_v. When passed a
 ``cl_platform_id``, an OpenCL|trade| platform is used to construct the
 platform. The ``cl_platform_id`` is retained and available via
 get_. When passed a :ref:`device_selector`, a platform is constructed
```

Comment:
Reference will be modified in "Device Selectors" section:
 `` :ref:`device_selector` ``, better to change to `` `device selector<ref:device-selector>`_ ``

Reply:
`` :ref:`device selector<device-selectors>` ``